src/bp/ui-studio/src/web/components/ChatEmulator/Emulator.jsx

```diff
@@ -225,11 +262,19 @@ export default class EmulatorChat extends React.Component {
     this.setState({ isTypingHidden: !this.state.isTypingHidden })
   }
 
+  hideSettings = () => this.setState({ isSettingsOpen: false })
+  displaySettings = () => this.setState({ isSettingsOpen: true })
+  updateSettings = newSettings => this.setState({ ...newSettings })
```

Comment:
is there a way to "clear settings" ?

Reply:
Not required. Creating a new session would just reset the user id to something else. The token is cleared when the page is reloaded